tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -34,12 +29,7 @@ class CompletionMatchSuite extends BaseCompletionSuite {
       |}""".stripMargin,
     """|match
        |match (exhaustive) Option[Int] (2 cases)
-       |""".stripMargin,
-    compat = Map(
-      "3" -> """|match
-                |match (exhaustive) Option (2 cases)
```

Comment:
We might need to adjust the results for nightly Scala version.

Reply:
I've ignored the tests for now, until we backport to Dotty.